exercises/reverse-string/description.md

```diff
@@ -0,0 +1,14 @@
+# Reverse String
+
+The introductory exercise to reverse an input string.
+
+This is a typical program for beginning programming in a new language
+or environment after being able to do simple tasks.
+
+The objectives are simple:
+
+- Write a function that reverses the input string.
+- Test various mixed case inputs and output the reverse of them.
+- Run the test suite and make sure that it succeeds.
+- Submit your solution and check it at the website.
+
```

Comment:
This description contains too many instructions.
The description should just describe what the problem is.

It would be fine to include some background on what a 'string' is.

Reply:
> It would be fine to include some background on what a 'string' is.

This is hard in a language agnostic way…

I think the most agnostic way were to say "its an ordered sequence of characters".

But even this description might fail if you accept BEAMs io-lists as strings…

Reply:
> This is hard in a language agnostic way…

I agree, which makes me think even including the word "string" is a bad idea.

maybe 'text' or 'words' or 'snippet' or 'sentence' or something else..?